### DISCUSSION TITLE
Width of Checkbox Controls

### DISCUSSION THREAD

**Discussion by nofue** [sqlpage/SQLPage#613] [Ideas]
Styling a form by using the width parameter works pretty well, however, checkboxes are smaller than other fields:

![Bildschirmfoto 2024-09-22 um 12 01 08](https://github.com/user-attachments/assets/24e35dce-eaa3-4be2-9514-1cdc9297a567)

Which looks sort of -hm-. 

And as we're at it: is there a chance to get a divider control which works in forms? 
And before I waste a lot of time trying it, is there (could there be) a "void" form item to allow for blank "fields" in the form? Or a new parameter which keeps other form controls from "flowing up" to the row above if it is short enough?

**Comment by lovasoa** [comment]
I think there was a problem with your screenshot

**Reply by nofue** [reply]
Not at all. I didn't want to show the actual controls, but just the way they are out of sync and also save some space on the receiving server. 
Explanation (if needed): The upper row of controls is date and text fields, the lower row is checkboxes. The difference in cumulated width can be seen at the right, where the checkboxes are way shorter than the other controls despite all having been set to 4 AS width …

**Comment by lovasoa** [comment]
Fixed in ce1b59f7d16566e4d504ad1231711ee9a84ee323 :)

Thanks for reporting it

**Reply by nofue** [reply]
You're welcome. Looking forward seeing it in action. Completed (well…) my first SQLPage app today, and it's great. It looks great, it's easy to build and it's snappy fast. It's not the most sophisticated thing in the world, just a replacement of an app I built with Drupal. But with SQLpage it is way nicer -- great job, thank you for this tool the world had to wait so long for.
